### PR TITLE
docs: servicedenuages.fr is domain parked, replace links with web.archive.org

### PR DESCRIPTION
**Pull request by adasauce**
so folks going through the repo and docs can find the relevant links and information.

- [x] Add documentation.
~~- [ ] Add tests.~~
~~- [ ] Add a changelog entry.~~
~~- [ ] Add your name in the contributors file.~~
~~- [ ] If you changed the HTTP API, update the [API_VERSION](https://github.com/Kinto/kinto/blob/master/kinto/__init__.py#L15) constant and add an API changelog entry [in the docs](https://github.com/Kinto/kinto/blob/master/docs/api/index.rst)~~
~~- [ ] If you added a new configuration setting, update the [`kinto.tpl`](https://github.com/Kinto/kinto/blob/master/kinto/config/kinto.tpl) file with it.~~

### CODE REVIEW

**Comment by Natim** [comment]
Hello @adasauce 

Thank you for your contribution. Could we consider using https://mozilla-services.github.io/servicedenuages.fr/ instead of webarchive?

Best regards,
Rémy

**Comment by adasauce** [comment]
sure thing @Natim that seems a lot cleaner.

File: docs/faq.rst [hunk 1]
```diff
@@ -75,7 +76,7 @@ Can I encrypt my data?
 
 Kinto server stores any data you pass to it, whether it's encrypted or not. We believe
 encryption should always be done on the client-side, and we make it `easy to use encryption in our Kinto.js client
-<http://www.servicedenuages.fr/en/kinto-encryption-example>`_.
+<https://web.archive.org/web/20171123111009if_/http://www.servicedenuages.fr/en/kinto-encryption-example>`_.
```
Comment (Natim) [inline docs/faq.rst:79]:
```suggestion
<https://mozilla-services.github.io/servicedenuages.fr/en/kinto-encryption-example>`_.
```

File: docs/tutorials/client-side-encryption.rst [hunk 2]
```diff
@@ -16,4 +16,4 @@ With Kinto, your records can have a text field that stores the encrypted attribu
 The client will encrypt the payload before sending to the server, and decrypt it upon reception.
 
 See the `online demo <https://michielbdejong.github.io/kinto-encryption-example/>`_ that leverages
-kinto.js and WebCrypto, and read `the complete tutorial <http://www.servicedenuages.fr/en/kinto-encryption-example>`_!
+kinto.js and WebCrypto, and read `the complete tutorial <https://mozilla-services.github.io/servicedenuages.fr/en/generic-storage-ecosystem>`_!
```
Comment (Natim) [inline docs/tutorials/client-side-encryption.rst:19]:
```suggestion
kinto.js and WebCrypto, and read `the complete tutorial <https://mozilla-services.github.io/servicedenuages.fr/en/kinto-encryption-example>`_!
```

**Comment by adasauce** [comment]
ah thanks for the cleanup, looks like I wasn't as careful the second time around.

**Pull request merged**